gradle/build-scan.gradle

```diff
@@ -1,10 +1,17 @@
 import org.elasticsearch.gradle.OS
+import org.gradle.initialization.BuildRequestMetaData
+
+import java.util.concurrent.TimeUnit
+
+
+long startTime = project.gradle.services.get(BuildRequestMetaData.class).getStartTime()
```

Comment:
This is probably overkill. I think just using `System.currentTimeMillis()` is a close enough approximate here as script evaluation time and build start time are mostly equivalent.

Reply:
I was under the impression that this will give me the time when the build started. 
I might have incorrectly assume that timestamps  are in utc so kibana will be ok with them.

Reply:
The timestamp may be UTC, but in either way Kibana _seems_ to treat it as local time.

Reply:
@mark-vieira the starting time for me is  `9:09:08` and Kibana does show TZ info on hover. I'm hoping it's converting that to your timezone for you.

Reply:
Looking at the links in the build scans generated for the PR checks, it seems to me that Kibana correctly shows the timestamps in the local timezone of the browser